File: 250317/코로나 메뉴얼/covid-manual.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String f_sym = sc.next();
        int f_tem = sc.nextInt();
        String s_sym = sc.next();
        int s_tem = sc.nextInt();
        String t_sym = sc.next();
        int t_tem = sc.nextInt();

        int count = 0;

        if(f_sym.charAt(0) == 'Y') {
            if(f_tem >= 37) {
                count++;
            }
        }
        if(s_sym.charAt(0) == 'Y') {
            if(s_tem >= 37) {
                count++;
            }
        }
        if(t_sym.charAt(0) == 'Y') {
            if(t_tem >= 37) {
                count++;
            }
        }

        if(count >= 2) {
            System.out.println("E");
        }
        else {
            System.out.println("N");
        }

    }
}

File: 250317/코로나 메뉴얼/covid-manual.cpp
#include <iostream>
using namespace std;

int main()
{
    char f_sym, s_sym, t_sym = ' ';
    int f_tem, s_tem, t_tem, count = 0;
    cin >> f_sym >> f_tem >> s_sym >> s_tem >> t_sym >> t_tem;

    if(f_sym == 'Y' && f_tem >= 37) {
        count++;
    }
    if(s_sym == 'Y' && s_tem >= 37) {
        count++;
    }
    if(t_sym == 'Y' && t_tem >= 37) {
        count++;
    }

    if(count >= 2) {
        cout << "E";
    }
    else {
        cout << "N";
    }
    return 0;
}